### PR TITLE
📈: Add a workflow to label pull requests

### PR DESCRIPTION
**Pull request by in-a-paddy**
## やったこと

- [x] Automatically add label to pull requests based on modified files.
  - `iOS`, `Android`, `app`, `build`, `chore`, `ci`, `doc`, `test`

## やっていないこと

Nothing

## 動作確認

- [x] This pull request is labeled as `chore` and `ci`

## デバイス

It's not about the devices.

## その他（レビュアーへの申し送り事項、懸念点など）

I can't think of anything.

### CODE REVIEW

File: .github/labeler.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+# Add 'Android' label to any change within the 'android' directory
+Android:
+  - android/**/*
+
+# Add 'app' label to any change within the 'app' directory and to the files listed below
+app:
+  - app/**/*
+  - app.json
+  - index.js
```
Comment (Takumon) [inline .github/labeler.yaml:13]: How about App.tsx?
I suppose this file should belong to `app` label because index.js labeled `app` imports this file.

**Review by Takumon — COMMENTED** [review]
Please check my comment.

File: .github/labeler.yaml (hunk 1 above)
Comment (in-a-paddy) [reply .github/labeler.yaml:13]: Let's place within `app` directory!

Comment (in-a-paddy) [reply .github/labeler.yaml:13]: Because it's easier to refer later if it's a separate pull request, I didn't move it in this one.

**Review by Takumon — APPROVED** [review]
OK!

**Pull request merged**